src/components/ChangePlan/Checkout/ContactInformation/ContactInformation.test.js

```diff
@@ -110,6 +113,12 @@ describe('Checkout component', () => {
 
       const inputPhone = container.querySelector('input#phone');
       fireEvent.change(inputPhone, { target: { value: '' } });
+
+      const selectSecurityQuestion = container.querySelector('select#securityquestion');
```

Comment:
In this part, you are checking what is the behavior `if security question field is empty` and `if answer field is empty`